examples_test.go

```diff
@@ -1,66 +1,65 @@
-package rollbar_test
```

Comment:
Could this remain `rollbar_test` so that anyone looking at this as a form of documentation could still  copy/paste from an example? Super minor since anyone writing go will know that the functions come from the `rollbar` module.

Reply:
ok I will change it before committing